en/docs/api-management/manage-api-traffic/subscribe-to-an-api-with-a-subscription-plan.md

```diff
@@ -11,6 +11,10 @@ To subscribe to an API with a subscription plan, follow the steps given below:
 5. In the **Subscription Management** pane that opens, click **+ Add APIs**.
 6. In the **Add APIs** pane that opens, select the API, API version, and subscription plan with which you want to subscribe to the API.
 7. Click **Add Subscription**.
+8. If the selected subscription plan requires approval, your subscription will be shown as "Pending" until it is approved by an organization admin or the API publisher.
```

Comment:
Shall we update step 7 as follows since the button name differs if approval enabled:

7. If the selected subscription plan requires administrator or API publisher approval to become active, click **Request Subscription**. Otherwise, click **Add Subscription**. If the selected subscription plan requires approval, your subscription will be shown as "Pending" until it is approved by an organization admin or the API publisher.

    !!! Note
         If the subscription plan requires administrator or API publisher approval to become active, the subscription status will be set to **Pending Creation** until it is reviewed and approved.